src/configure-apollo-network.js

```diff
@@ -10,12 +10,27 @@
  * Requests will automatically timeout after 1000ms, unless another
  * timeout is provided via a 'timeout' property.
  */
-import * as ApolloClient from "apollo-client";
+import * as ApolloClientModule from "apollo-client";
```

Comment:
Oof - that's kind of annoying!

Reply:
We need the Node version of the module to run inside the browser context, so we pass the whole module in this way. Seemed clearer to have this be named as the module than have it clash with the name of the type within the module too.